collect_app/src/main/java/org/odk/collect/android/utilities/ExternalWebPageHelper.java

```diff
@@ -14,7 +16,8 @@
 /**
  * Created by sanjeev on 17/3/17.
  */
-public class CustomTabHelper {
+public class ExternalWebPageHelper {
```

Comment:
It felt like this was a more appropriate name given it falls back from custom tabs